package.json

```diff
@@ -28,7 +28,8 @@
   "homepage": "https://github.com/zattoo/changelog#readme",
   "dependencies": {
     "@actions/core": "1.2.4",
-    "@actions/github": "4.0.0"
+    "@actions/github": "4.0.0",
+    "glob-promise": "3.4.0"
```

Comment:
why not https://github.com/isaacs/node-glob ?

Reply:
This one uses glob internally, with normal glob you have to use callbacks this is just a wrapper to use promises.

Reply:
using origin lib with callback is totally fine, depending on proxy lib for syntax sugar is so so

Reply:
Refactored now to use node-glob and `util.promisify`.